### ISSUE TITLE
evaluate scllvm memcpy and memset performace

### ISSUE THREAD

**Issue by bigwad** [Starcounter/Starcounter#3917]
Now we use llvm built-in implementations.
cc @alemoi

**Comment by alemoi** [comment]
Yes, this was some quick find that I made by looking for embedded cross-platform functionality inside LLVM itself. Not sure about their performance properties, but its, as we discussed, used in non-critical code places.
